src/add_compute.jl

```diff
@@ -222,7 +222,7 @@ function add_compute!(
     # instr = instruction(first(ex.args))::Symbol
     instr = instruction!(ls, first(ex.args))::Instruction
     args = @view(ex.args[2:end])
-    (instr.instr === :(^) && length(args) == 2 && (args[2] isa Number)) && return add_pow!(ls, var, args[1], args[2], elementbytes, position)
+    (instr.instr === :(^) && length(args) == 2 && (args[2] isa Number)) && return add_pow!(ls, var, args[1], args[2]::Number, elementbytes, position)
```

Comment:
Would it be better to extract `args[2]` and then check that the extracted value is a number, rather than index twice?

Reply:
done